Shared/Views/DonorChart.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  DonorChart.swift
+//  St Jude
+//
+//  Created by Ben Cardy on 17/09/2022.
+//
+
+import SwiftUI
+import Charts
+
+struct DonorChartValue: Identifiable {
+    let id: Int
+    let amount: Double
+    let date: Date
+}
+
+struct DonorChart: View {
+    
+    let donations: [TiltifyDonorsForCampaignDonation]
+    let total: TiltifyAmount
+    
+    @State private var maxValue: Double = 0
+    @State private var minValue: Double = 0
+    @State private var showDonationValues: Bool = false
+    @State private var chartValues: [DonorChartValue] = []
+    
+    var body: some View {
+        if #available(iOS 16.0, *) {
```

Comment:
I'd have just annotated the entire struct, tbh

Reply:
Yeah that's fair 😅